bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieProtocol.java

```diff
@@ -62,6 +62,16 @@
      */
     public static final int MASTER_KEY_LENGTH = 20;
 
+    /**
```

Comment:
are you sure to support ledger type in v2 protocol? if you want to support it, I would suggest you first make sure v2 and v3 support same semantic first before introducing ledger type.